kalkulator/src/main/java/no/nav/folketrygdloven/kalkulator/steg/refusjon/AvklaringsbehovutlederVurderRefusjon.java

```diff
@@ -16,23 +17,45 @@ private AvklaringsbehovutlederVurderRefusjon() {
             throw new IllegalStateException("Har ikke korrekt input for å vurdere aksjsonspunkt i vurder_refusjon steget");
         }
-	    var vurderInput = (VurderRefusjonBeregningsgrunnlagInput) input;
-	    var orginaltBGGrunnlag = vurderInput.getBeregningsgrunnlagGrunnlagFraForrigeBehandling();
-        if (orginaltBGGrunnlag.isEmpty() || orginaltBGGrunnlag.stream().noneMatch(gr -> gr.getBeregningsgrunnlagHvisFinnes().isPresent())) {
+
+        if (erFPEllerSVP(vurderInput) && vurderInput.isEnabled("refusjonsfrist.flytting", false)
+            && skalHaAvklaringsbehovVurderRefusjonskravKommetForSent(vurderInput)) {
+            return true;
+        }
+
+        var forrigeGrunnlagListe = vurderInput.getBeregningsgrunnlagGrunnlagFraForrigeBehandling().stream()
+            .flatMap(gr -> gr.getBeregningsgrunnlagHvisFinnes().stream())
+            .toList();
+
+        if (forrigeGrunnlagListe.isEmpty()) {
             return false;
         }
+
+        return harAndelerMedØktRefusjonIUtbetaltPeriode(input, periodisertMedRefusjonOgGradering, forrigeGrunnlagListe);
+    }
+
+    private static boolean harAndelerMedØktRefusjonIUtbetaltPeriode(BeregningsgrunnlagInput input,
+                                                                    BeregningsgrunnlagDto periodisertMedRefusjonOgGradering,
+                                                                    List<BeregningsgrunnlagDto> forrigeGrunnlagListe) {
         var perioderTilVurderingTjeneste = new PerioderTilVurderingTjeneste(input.getForlengelseperioder(), periodisertMedRefusjonOgGradering);
         var grenseverdi = periodisertMedRefusjonOgGradering.getGrunnbeløp().multipliser(KonfigTjeneste.getAntallGØvreGrenseverdi());
-        var orginaleBG = orginaltBGGrunnlag.stream().flatMap(gr -> gr.getBeregningsgrunnlagHvisFinnes().stream())
-                .collect(Collectors.toList());
-	    var andelerMedØktRefusjonIUtbetaltPeriode = orginaleBG.stream()
-                .flatMap(originaltBg -> AndelerMedØktRefusjonTjeneste.finnAndelerMedØktRefusjon(periodisertMedRefusjonOgGradering, originaltBg, grenseverdi, input.getYtelsespesifiktGrunnlag()).entrySet().stream())
-                .filter(e -> perioderTilVurderingTjeneste.erTilVurdering(e.getKey()))
-                .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));
-
-        return !andelerMedØktRefusjonIUtbetaltPeriode.isEmpty();
+        return forrigeGrunnlagListe.stream()
+            .flatMap(
+                forrigeGrunnlag -> AndelerMedØktRefusjonTjeneste.finnAndelerMedØktRefusjon(periodisertMedRefusjonOgGradering, forrigeGrunnlag, grenseverdi,
+                    input.getYtelsespesifiktGrunnlag()).entrySet().stream())
+            .anyMatch(e -> perioderTilVurderingTjeneste.erTilVurdering(e.getKey()));
+    }
+
+    private static boolean erFPEllerSVP(VurderRefusjonBeregningsgrunnlagInput vurderInput) {
+        return vurderInput.getFagsakYtelseType() == FagsakYtelseType.FORELDREPENGER
```

Comment:
Vi pleier som regel å bruke .equals for å sjekke likhet mellom objekter, men siden det er enum funker det jo fint.